src/main/kotlin/uk/gov/gdx/datashare/service/EventDataService.kt

```diff
@@ -161,7 +161,7 @@ data class EventNotification(
   val eventType: String,
   @Schema(description = "ID from the source of the notification", required = true, example = "999999901")
   val sourceId: String,
-  @Schema(description = "Event Data", required = false)
+  @Schema(description = "Event Data - only returned for thick events, otherwise this field will be empty.  Full dataset for the event can be obtained by calling /events/{id}", required = false)
```

Comment:
```suggestion
  @Schema(description = "Event Data - only returned when the consumer has Full data is only returned when the consumer has `enrichmentFieldsIncludedInPoll` enabled, otherwise this field will be empty.  Full dataset for the event can be obtained by calling /events/{id}", required = false)
```